carbonmark/components/ExitModal/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { t, Trans } from "@lingui/macro";
+import AccountBalanceWalletIcon from "@mui/icons-material/AccountBalanceWallet";
+import LaunchOutlinedIcon from "@mui/icons-material/LaunchOutlined";
+import SpaceDashboardIcon from "@mui/icons-material/SpaceDashboard";
+import { CarbonmarkButton } from "components/CarbonmarkButton";
+import { Modal, ModalProps } from "components/shared/Modal";
+import * as styles from "./styles";
+
+type ExitModalProps = Omit<ModalProps, "children"> & { retireLink: string };
+
+export const ExitModal = (props: ExitModalProps) => (
+  <Modal {...props} className={styles.modal}>
+    <div className={styles.content}>
+      <div>
+        <LaunchOutlinedIcon />
+        <Trans>
+          You will be taken to KlimaDAO.finance to complete this transaction.
+        </Trans>
+      </div>
+      <div>
+        <AccountBalanceWalletIcon />
+        <Trans>
```

Comment:
Custom message IDs are useful for these really long strings to avoid duplicate data and inflating our string file sizes over time, as you can see in the output:

(short strings I prefer with no ids just to avoid boilerplate and duplicates)

![image](https://user-images.githubusercontent.com/88635679/235948045-48928184-6a9b-45f8-aa7d-ad7378ea95cd.png)

Reply:
Noted & sounds good! Will update this to include custom message IDs for these longer strings.